vms/traffic-gen/scripts/customize-vm

```diff
@@ -59,7 +59,6 @@ setup_boot_service() {
   local service_name="dpdk-checkup-boot.service"
   local checkup_boot_script_full_path="/usr/bin/dpdk-checkup-boot.sh"
   local checkup_boot_service_full_path="/usr/lib/systemd/system/$service_name"
-  local checkup_tuned_adm_set_marker_full_path="/var/dpdk-checkup-tuned-adm-set-marker"
```

Comment:
This whole section is later removed, please consider dropping this commit.

Reply:
in order for me to perform a perfect "copy" as said in the commit - I need this commit done first